lesson_06/homework_06_2.py

```diff
@@ -0,0 +1,8 @@
+#Напишіть цикл, який буде вимагати від користувача ввести слово, 
+# #в якому є літера "h" (враховуються як великі так і маленькі). 
+# Цикл не повинен завершитися, якщо користувач ввів слово без букви "h".
+user_input = input("Enter a word: ")
+while "h" not in user_input or "H" not in user_input:
+    user_input = input("Enter a word with the letter 'h' or 'H': ")
```

Comment:
```suggestion
while "h" not in user_input or "H" not in user_input:
    user_input = input("Enter a word with the letter 'h' or 'H': ")
```
у вас виходить завершення циклу лише коли є і 'h' і 'H'
![зображення](https://github.com/user-attachments/assets/26db0efa-abc2-4ca8-8d00-f09d55880d4e)

Reply:
Дякую за зауваження, виправила.